atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/SchemaMutationLock.java

```diff
@@ -253,31 +276,62 @@ private void waitForSchemaMutationLockWithoutCas() throws TimeoutException {
     }
 
     private void schemaMutationUnlock(long perOperationNodeId) {
+        boolean unlockDone = false;
+        boolean isInterrupted = false;
+        long currentSleepMillis = TIME_BETWEEN_LOCK_ATTEMPT_ROUNDS_MILLIS;
+        for (int unlockRetryCount = 0; unlockRetryCount < MAX_UNLOCK_RETRY_COUNT; unlockRetryCount++) {
+            unlockDone = trySchemaMutationUnlockOnce(perOperationNodeId);
+            if (unlockDone) {
+                break;
+            }
+
+            currentSleepMillis = Math.min(2 * currentSleepMillis, TIME_BETWEEN_LOCK_ATTEMPT_ROUNDS_MILLIS_CAP);
+            try {
+                Thread.sleep(currentSleepMillis);
+            } catch (InterruptedException e) {
+                isInterrupted = true;
```

Comment:
That makes sense. 

I'm not convinced that ignoring the interrupt is best practice (instead of, for example, throwing with something like "Unable to unlock because we were interrupted" or even retrying without the wait as a fast, last-ditch retry attempt).  To be fair, I don't have a clear notion of why an interrupt might occur here - so if you've thought this through and are convinced the current approach is the right way to go, I'm fine for this to merge as is.